docs/setup.rst

```diff
@@ -70,6 +70,11 @@ The following additional settings are recommended:
     ``"zyte_crawlers.middlewares.CrawlingLogsMiddleware": 1000``, to log crawl
     data in JSON format for debugging purposes.
 
+-   Update :setting:`SPIDER_MIDDLEWARES <scrapy:SPIDER_MIDDLEWARES>` to include
+    ``"zyte_crawlers.middlewares.AllowOffsiteMiddleware": 500`` and remove
+    ``"scrapy.spidermiddlewares.offsite.OffsiteMiddleware"``. This allows for
+    crawling item links outside of the domain.
```

Comment:
This being a default middleware, it needs to be disabled, not “removed”, which might not be clear for some users. Maybe…

```suggestion
-   Update :setting:`SPIDER_MIDDLEWARES <scrapy:SPIDER_MIDDLEWARES>` to include
    ``"zyte_spider_templates.middlewares.AllowOffsiteMiddleware": 500`` and
    ``"scrapy.spidermiddlewares.offsite.OffsiteMiddleware": None``. This allows for
    crawling item links outside of the domain.
```